protocols/imap/src/test/java/org/apache/james/imap/processor/base/UidMsnMapperTest.java

```diff
@@ -218,6 +218,45 @@ public void removeShouldKeepAValidMappingWhenDeletingMiddle() {
     }
 
     @Test
+    public void addUidShouldCommute() {
```

Comment:
I don't want a definition of commute, I asked what did you meant.
In this case, I just can't map a and b so I don't think it's a commutation.